mgmtfn/k8splugin/kubeClient_test.go

```diff
@@ -367,8 +378,16 @@ func setupTestServer(c *C) {
 	tlsCfg := &tls.Config{
 		Certificates: []tls.Certificate{cert},
 		RootCAs:      caPool,
+		ClientAuth:   tls.VerifyClientCertIfGiven,
+		ClientCAs:    caPool,
 	}
 	tlsCfg.BuildNameToCertificate()
+	// Read the token
+	var contivK8Config ContivConfig
+
+	err = getConfig(testCfgFile, &contivK8Config)
+	token = contivK8Config.K8sToken
+
```

Comment:
@neelimamukiri: If understood this right, the test code will now use certs and token always. Can we add a simple test to cover these two scenarios: 1) auth with certs only 2) auth with token only.